packages/reports/src/utils/consts/cms.js

```diff
@@ -81,7 +81,7 @@ const BLOCK_CONTENT_TYPES = {
   AUTHOR: "author",
   FOOTNOTE: "footnote",
   PARAGRAPH: "paragraph",
-  SELECTOR: "selector",
+  SELECTOR: "selector", // TODO - move to logic types
```

Comment:
Let's either:
1) change this to `todo1.0` which has been my standard so far
or, if `TODO` is a thing that is supported by github/typescript/vscode/your own heart
2) change all `todo1.0`'s to `TODO` and stick with that from now on.